tests/2d_examples/test_2d_taylor_green_euler/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${SPHINXSYS_PROJECT_DIR}/cmake) # main (top) cmake dir
+
+set(CMAKE_VERBOSE_MAKEFILE on)
+
+STRING(REGEX REPLACE ".*/(.*)" "\\1" CURRENT_FOLDER ${CMAKE_CURRENT_SOURCE_DIR})
+PROJECT("${CURRENT_FOLDER}")
+
+include(ImportSPHINXsysFromSource_for_2D_build)
+
+SET(EXECUTABLE_OUTPUT_PATH "${PROJECT_BINARY_DIR}/bin/")
+SET(LIBRARY_OUTPUT_PATH ${PROJECT_BINARY_DIR}/lib)
+SET(BUILD_INPUT_PATH "${EXECUTABLE_OUTPUT_PATH}/input")
+SET(BUILD_RELOAD_PATH "${EXECUTABLE_OUTPUT_PATH}/reload")
+
+file(MAKE_DIRECTORY ${BUILD_INPUT_PATH})
+execute_process(COMMAND ${CMAKE_COMMAND} -E make_directory ${BUILD_INPUT_PATH})
+file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/regression_test_tool/regression_test_tool.py
+        DESTINATION ${BUILD_INPUT_PATH})
+file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/regression_test_tool/WaterBody_TotalMechanicalEnergy_runtimes.dat
+        DESTINATION ${BUILD_INPUT_PATH})
+file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/regression_test_tool/WaterBody_MaximumSpeed_runtimes.dat
+        DESTINATION ${BUILD_INPUT_PATH})
+
+aux_source_directory(. DIR_SRCS)
+ADD_EXECUTABLE(${PROJECT_NAME} ${DIR_SRCS})
+
+#add_test(NAME ${PROJECT_NAME} COMMAND ${PROJECT_NAME} --i=true
+#        WORKING_DIRECTORY ${EXECUTABLE_OUTPUT_PATH})
+
+if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
+        set_target_properties(${PROJECT_NAME} PROPERTIES VS_DEBUGGER_WORKING_DIRECTORY "${EXECUTABLE_OUTPUT_PATH}")
+        target_link_libraries(${PROJECT_NAME} sphinxsys_2d ${TBB_LIBRARYS} ${Simbody_LIBRARIES})
+        add_dependencies(${PROJECT_NAME} sphinxsys_2d)
+else(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
+        if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+                target_link_libraries(${PROJECT_NAME} sphinxsys_2d ${TBB_LIBRARYS} ${Simbody_LIBRARIES} ${Boost_LIBRARIES} stdc++)
+        else(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+                target_link_libraries(${PROJECT_NAME} sphinxsys_2d ${TBB_LIBRARYS} ${Simbody_LIBRARIES} ${Boost_LIBRARIES} stdc++ stdc++fs)
+        endif(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+endif(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
```

Comment:
Why is this file new

Reply:
This is due to error of merging, should be deleted.

Reply:
This is why we suggest starting a clean PR - it's error-prone at the moment, and very hard to review

Reply:
> This is why we suggest starting a clean PR - it's error-prone at the moment, and very hard to review

I will close this PR, and request a new one.